package.json

```diff
@@ -155,7 +155,7 @@
     "typescript": "~4.6.0"
   },
   "engines": {
-    "node": ">= 12.*"
+    "node": ">= 14.*"
```

Comment:
I don't think we can bump this in a point release since it would be a breaking change that would be very unexpected. Bumping the version for a particular CI job should be fine though I am curious why it failed: theoretically the smoke test should work on 4.4 without changes.

Reply:
Isn't the rule about releases that each release supports whatever node version is still under maintenance?

Reply:
We discussed on discord, but writing here as well: I think a node version bump in a patch release would be very surprising to users. The policy states only when we drop support on the primary branch of ember-cli https://github.com/ember-cli/ember-cli/blob/master/docs/node-support.md